docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -404,12 +477,14 @@ Standard ONNX\* operators:
 | Acosh | No |
 | Add | No |
 | Affine | No |
+| And | No |
 | ArgMax | No |
 | ArgMin | No |
 | Asin | No |
 | Asinh | No |
 | Atan | No |
 | Atanh | No |
+| ATen | No |
```

Comment:
Supported for specific value of "operator" attribute. To be precise for "embedding_bag" only. Refer to model-optimizer/extensions/front/ATenToEmbeddingBag.py